### PR TITLE
[Wallet] Implement remaning UI for new verification flow

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Add new icons
- Implement verification interstitial screen
- Implement verification input screen
- Style tweaks
- Add hitslop to paste / clear icons to make them easier to target

Still TODO: Tie the screens into the verification saga. Will do that in the next PR

### Tested

Relevant screens tested locally on Android
still TODO test iOS, will do after done verification implementation

### Related issues

- For #1523 

### Backwards compatibility

Yes

wa-ver-icons
wa-ver-interstitial
wa-ver-input

### CODE REVIEW

File: packages/walletkit/src/attestations.ts [hunk 1]
```diff
@@ -182,12 +182,12 @@ export function messageContainsAttestationCode(message: string) {
 
 export function extractAttestationCodeFromMessage(message: string) {
```
Comment (cmcewen) [inline packages/walletkit/src/attestations.ts:183]: i feel that we are a little inconsistent with throwing vs nulls - wdyt?

Comment (jmrossy) [reply packages/walletkit/src/attestations.ts:183]: Agreed. I'm generally in favor of exceptions but nulls were helpful here. Hbu WDYT?

File: packages/mobile/src/verify/VerificationInputScreen.tsx [hunk 2]
```diff
@@ -1,23 +1,181 @@
...
+          />
+          <Link style={styles.missingCodesLink} onPress={this.onPressCodesNotReceived}>
+            {t('input.codesMissing')}
+          </Link>
         </ScrollView>
+        <Modal isVisible={isModalVisible}>
+          <View style={styles.modalContainer}>
+            <View style={styles.modalTimerContainer}>
+              <LoadingSpinner />
+              <Text style={fontStyles.body}>{'0:49'}</Text>
```
Comment (jeanregisser) [inline packages/mobile/src/verify/VerificationInputScreen.tsx:165]: I assume the real value will be added in the next PR?

File: packages/mobile/src/verify/VerificationInterstitialScreen.tsx [hunk 3]
```diff
@@ -1,19 +1,45 @@
...
 import { withNamespaces, WithNamespaces } from 'react-i18next'
-import { ScrollView, StyleSheet, Text } from 'react-native'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
 import SafeAreaView from 'react-native-safe-area-view'
+import { isE2EEnv } from 'src/config'
 import { Namespaces } from 'src/i18n'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+
+const SCREEN_DUARTION = 5000 // 5s
```
Comment (jeanregisser) [inline packages/mobile/src/verify/VerificationInterstitialScreen.tsx]: typo

File: packages/mobile/src/verify/VerificationInterstitialScreen.tsx [hunk 4]
```diff
@@ -1,19 +1,45 @@
...
+)
 
 class VerificationInterstitialScreen extends React.Component<WithNamespaces> {
   static navigationOptions = null
 
+  componentDidMount() {
+    setTimeout(() => {
+      navigate(Screens.VerificationInputScreen)
+    }, SCREEN_DUARTION)
+  }
```
Comment (jeanregisser) [inline packages/mobile/src/verify/VerificationInterstitialScreen.tsx]: Can this screen be unmounted before the timer triggers?

**Review by jeanregisser — APPROVED** [review]
Looking good 👍

File: packages/mobile/src/verify/VerificationInputScreen.tsx (hunk 2 above)
Comment (jmrossy) [reply packages/mobile/src/verify/VerificationInputScreen.tsx:165]: Yes, already done in my latest

File: packages/mobile/src/verify/VerificationInterstitialScreen.tsx (hunk 3 above)
Comment (jmrossy) [reply packages/mobile/src/verify/VerificationInterstitialScreen.tsx]: Will fix thanks!

File: packages/mobile/src/verify/VerificationInterstitialScreen.tsx (hunk 4 above)
Comment (jmrossy) [reply packages/mobile/src/verify/VerificationInterstitialScreen.tsx]: I'll clear the timeout just in case, thanks!

**Pull request merged**